src/scripts/cleanfeed.css

```
@@ -12,8 +12,10 @@
   display: flex;
   justify-content: center;
   align-items: center;
+  text-align: center;
   height: 100%;
   width: 100%;
+  line-height: 1;
```

This is _sort of_ a nitpick but I might as well require this since releases are slow...

`line-height: 1.5` is the standard for readability for "regular" sized text:

```suggestion
  line-height: 1.5;
```